### DISCUSSION TITLE
Cast variable from string to float

### DISCUSSION THREAD

**Discussion by marcoberri** [SigNoz/signoz#6271] [Q&A] [answered]
Hi,
I have a variable in dashboard that extract the response status code from traces table:

![image](https://github.com/user-attachments/assets/1ffc101e-3c7f-4fb4-a2b9-2895e6f69135)

when i try add the variable to the query builder i see this:

![image](https://github.com/user-attachments/assets/1fee9402-4f3b-4f43-a106-7e11e353552c)

is possible cast a variable to float in variable dasboard definition or query builder?

tnx.

**Comment by srikanthccv** [comment, marked as answer]
It's a bug. We will work on fixing it.
